libs/dh/wholesale/feature-settlement-reports/src/lib/drawer/dh-settlement-report-drawer.component.ts

```diff
@@ -73,21 +77,36 @@ import { DhSettlementReportsStatusComponent } from '../util/dh-settlement-report
         margin: var(--watt-space-ml);
         width: 60%;
       }
+
+      .card-grid-areas {
+        margin: var(--watt-space-ml);
+      }
     `,
   ],
   templateUrl: './dh-settlement-report-drawer.component.html',
 })
 export class DhSettlementReportDrawerComponent {
   drawer = viewChild.required<WattDrawerComponent>(WattDrawerComponent);
 
+  tableSource = new WattTableDataSource<DhSettlementReport['gridAreas'][0]>();
```

Comment:
Suggestion: Extract this to its own type for better readability

Reply:
This can also be a `string`